frontend/src/components/TournamentMatch.js

```diff
@@ -833,12 +833,22 @@ const TournamentMatch = props => {
                   </button>
                 </div>
                 <div class="space-y-6 p-6">
-                  <MatchSpiritScoreForm
-                    match={props.match}
-                    tournamentSlug={props.tournamentSlug}
-                    oppTeamNo={isTeamAdminOf(props.match["team_1"].id) ? 2 : 1}
-                    curTeamNo={isTeamAdminOf(props.match["team_1"].id) ? 1 : 2}
-                  />
+                  <Show when={isTeamAdminOf(props.match["team_1"].id)}>
+                    <MatchSpiritScoreForm
+                      match={props.match}
+                      tournamentSlug={props.tournamentSlug}
+                      oppTeamNo={2}
+                      curTeamNo={1}
+                    />
+                  </Show>
+                  <Show when={isTeamAdminOf(props.match["team_2"].id)}>
+                    <MatchSpiritScoreForm
+                      match={props.match}
+                      tournamentSlug={props.tournamentSlug}
+                      oppTeamNo={1}
+                      curTeamNo={2}
+                    />
+                  </Show>
```

Comment:
So if a person is admin of both teams, the spirit score form will be a huge form right ? Would it be better to have 2 spirit-score-submit buttons here, one for each team ?

Reply:
yeah but then i thought there will be lot of buttons.. and buttons also will be big as we need to mention for which team the spirit button is for

Reply:
correct, there might be too many buttons. Yeah and team names can get long too, hmmm

Reply:
ok, lets leave it like this for now, let's hear admins next tournament if there were any problems.